src/admin-views/settings/emails/email-template/body/add-links.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+
+
+?>
+<tr>
+	<td style="padding:0;">
+		<table role="presentation" style="width:100%;border-collapse:collapse;border:0;border-spacing:0;">
+			<tr>
+				<td style="padding:30px 10px;text-align:right;width:50%" align="right">
+					<a href="#" style="color:#3C434A">
+						Add event to iCal
+					</a>
+				</td>
+				<td style="padding:30px 10px;text-align:left;width:50%" align="left">
+					<a href="#" style="color:#3C434A">
+						Add event to Google Calendar
```

Comment:
Let's make strings translatable. (even though this one is going to be probably added in TEC)